cfgov/v1/__init__.py

```diff
@@ -51,6 +51,11 @@ def strip_accents(value):
     return only_ascii
 
 
+def date_formatter(dt, text_format=False):
```

Comment:
More of a note to @chosak @richaagarwal @hillaryj @rosskarchner @higs4281 and myself than actionable in this PR, but it'd be nice if our custom filters had a place to live that wasn't `__init__.py`.

Reply:
@willbarton I wasn't sure if I was putting this in the right place -- I'd be happy to move it if there's a better home for filters... :)

Reply:
What about doing [something like this](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L161), i.e. add a new `v1.jinja2tags.something`, and add it to settings. That should include it globally without needing to add it to the monster list of things we stick into the environment in `__init__.py`. The unit tests for this could live alongside [these here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/tests/test_jinja2tags.py).

Reply:
@chosak I updated to use the `jinja2tags` approach in https://github.com/cfpb/cfgov-refresh/pull/3680/commits/4b82e30488cceb7f5ccdb0656b320e87647b56b9 -- thanks for the suggestion!